styles/styles.css

```diff
@@ -238,6 +242,59 @@ main img {
     width: 47%;
 }
 
+body.article div.section {
+    background-color: var(--background-color);
+}
+
+body.article div.article-hero p {
+    margin: 0.85em 0;
+    color: var(--link-color);
+    font-size: var(--body-font-size-xs);
+    font-weight: bold;
+}
+
+body.article .columns > div > div:first-of-type > p {
+    font-size: var(--body-font-size-xs);
+    margin: 0.85em 0;
+    font-weight: bold;
+}
+
+body.article div.article-hero h1 {
+    font-size: 60px;
+}
+
+body.article div.article-body {
+    font-size: var(--body-font-size-xs);
+}
+
+body.article div.article-body li strong {
```

Comment:
you may need to add the same thing for selector `article div.article-body p strong`

Reply:
we'll open another issue/pr if needed.